### PR TITLE
adding registerAppFeature action to SDKInfo plugin

### PR DESCRIPTION
**Pull request by ivanbogdanov**

### CODE REVIEW

**Comment by ivanbogdanov** [comment]
Added unregister code and ran `tools/update.sh`

File: libs/smartsync.js [hunk 1]
```diff
@@ -80,6 +80,12 @@
             smartstoreClient = cordova.require("com.salesforce.plugin.smartstore.client");
         }
 
+        var sdkinfo = cordova.require("com.salesforce.plugin.sdkinfo");
+        if(sdkinfo && typeof(sdkinfo.registerAppFeature) === 'function'){
+          // register SmartSync JS App feature
+          cordova.require("com.salesforce.plugin.sdkinfo").registerAppFeature('SJ')
```
Comment (bhariharan) [inline libs/smartsync.js]: This can be `sdkinfo` too.

Comment (ivanbogdanov) [inline libs/smartsync.js]: Uh. Right

**Comment by ivanbogdanov** [comment]
Fixed

File: libs/smartsync.js [hunk 2]
```diff
@@ -80,6 +80,12 @@
             smartstoreClient = cordova.require("com.salesforce.plugin.smartstore.client");
         }
 
+        var sdkinfo = cordova.require("com.salesforce.plugin.sdkinfo");
```
Comment (wmathurin) [inline libs/smartsync.js]: We test/debug sample apps in browsers (and some external developers have also used smartsync.js in web apps). So the code should be safe outside a container.
Could you do the following instead:

```javascript
if (window.cordova && window.cordova.require("com.salesforce.plugin.sdkinfo")) {
    var sdkinfo = cordova.require("com.salesforce.plugin.sdkinfo");
    if (typeof(sdkinfo.registerAppFeature) === 'function') sdksdkinfo.registerAppFeature('SJ');
 }
```
```

**Pull request merged**